pkg/apis/servicecatalog/types.go

```diff
@@ -184,9 +194,12 @@ type Instance struct {
 // InstanceSpec represents a description of an Instance.
 type InstanceSpec struct {
 	// ServiceClassName is the reference to the ServiceClass this is an
-	// instance of.  Immutable.
+	// instance of.
+	//
+	// Immutable.
```

Comment:
The commented newline I like because I've seen more than one formatter that is likely to smash that single word back into the previous sentence like it is. 

The intent of "immutable" on it's own line is a declaration that stands out.